flatpak-builder/index.js

```diff
@@ -394,7 +394,7 @@ const run = async (config) => {
       const artifactClient = artifact.create()
       core.info('Uploading artifact...')
       // Append the arch to the bundle name to prevent conflicts in multi-arch jobs
-      const bundleName = config.bundle.replace('.flatpak', '') + `-${config.arch}`
+      const bundleName = config.bundle.replace('.flatpak', '') + `-flatpak-${config.arch}`
```

Comment:
Instead, re-add the .flatpak suffix here

Reply:
Done